code/game/objects/effects/landmarks/landmarks.dm

```diff
@@ -413,3 +413,34 @@
 	GLOB.lz2_shuttle_console_turfs_list += loc
 	return INITIALIZE_HINT_QDEL
 
+/obj/effect/landmark/fob_sentry
+	name = "Fob sentry"
+	icon = 'icons/Marine/sentry.dmi'
+	icon_state = "sentry"
+
+/obj/effect/landmark/fob_sentry/Initialize()
+	. = ..()
+	GLOB.fob_sentries += loc
+	return INITIALIZE_HINT_QDEL
+
+/obj/effect/landmark/fob_sentry_rebel
+	name = "Rebel fob sentry"
+	icon = 'icons/Marine/sentry.dmi'
+	icon_state = "sentry"
+
+/obj/effect/landmark/fob_sentry_rebel/Initialize()
+	. =	..()
+	GLOB.fob_sentries_rebel += loc
+	return INITIALIZE_HINT_QDEL
+
+/obj/effect/landmark/sensor_tower
+	name = "Sensor tower"
+	icon = 'icons/obj/structures/sensor.dmi'
+	icon_state = "sensor"
+
+/obj/effect/landmark/sensor_tower/Initialize()
+	. =	..()
```

Comment:
These two still use tab instead of a space

Reply:
aaarf